X/Xorg_xcb_util_image/build_tarballs.jl

```diff
@@ -3,23 +3,27 @@
 using BinaryBuilder
 
 name = "Xorg_xcb_util_image"
-version = v"0.4.0"
+version = v"0.4.1"
 
 # Collection of sources required to build libxcb
 sources = [
-    ArchiveSource("https://xcb.freedesktop.org/dist/xcb-util-image-$(version).tar.bz2",
-                  "2db96a37d78831d643538dd1b595d7d712e04bdccf8896a5e18ce0f398ea2ffc"),
+    ArchiveSource("https://xcb.freedesktop.org/dist/xcb-util-image-$(version).tar.xz",
+                  "ccad8ee5dadb1271fd4727ad14d9bd77a64e505608766c4e98267d9aede40d3d"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/xcb-util-image-*/
 CPPFLAGS="-I${prefix}/include"
+LDFLAGS="-L${prefix}/lib"
 
 # When compiling for things like ppc64le, we need newer `config.sub` files
 update_configure_scripts
 
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --enable-malloc0returnsnull=no
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} \
+    --enable-malloc0returnsnull=no \
+    --with-pic \
+    PKG_CONFIG_PATH="${prefix}/lib/pkgconfig:${prefix}/share/pkgconfig"
```

Comment:
Why rewriting the [default value](https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/45acaa5c78a091ef75e5daa5de7a8676600509a5/src/Runner.jl#L1316)?

Reply:
The relevant option that fixed compilation was `--with-pic` here.